plugin/src/main/java/org/jenkinsci/plugins/workflow/cps/CpsFlowExecution.java

```diff
@@ -548,11 +566,15 @@ public void run() {
              * During sandbox execution, we need to call sandbox interceptor while executing asynchronous code.
              */
             private Env createInitialEnv() {
-                return Envs.empty( isSandbox() ? new SandboxInvoker() : new DefaultInvoker());
+                return Envs.empty(createInvoker());
             }
         });
     }
 
+    Invoker createInvoker() {
+        return LoggingInvoker.create(isSandbox());
```

Comment:
Maybe worth adding a system property to bypass `LoggingInvoker` in case there is an issue.